docs/releasing/publishing-from-a-support-branch.md

```diff
@@ -0,0 +1,162 @@
+
+If the `main` branch contains unreleased changes that are releasable immediately, [follow the steps in Publishing GOV.UK Frontend](/docs/releasing/publishing.md) instead.
+
+If the `main` branch only has a few unreleasable changes and these are easy to revert, you can do the following:
+
+1. Revert the unreleasable changes on the `main` branch
+2. Follow the steps to publish GOV.UK Frontend
+3. Add the reverted changes to the `main` branch
+
+However, this approach has risks. For example, it creates a messy commit history on the `main` branch.
+
+## Release GOV.UK Frontend from support branch
+
+### Before you release GOV.UK Frontend
+
+1. At stand up, tell the GOV.UK Design System team we are close to releasing.
+
+2. Developers to raise new issues in the team GitHub repositories ([govuk-frontend](https://github.com/alphagov/govuk-frontend), [govuk-frontend-docs](https://github.com/alphagov/govuk-frontend-docs), [govuk-prototype-kit](https://github.com/alphagov/govuk-prototype-kit)) to:
+    - create announcement draft for the new release (example card: [#2108](https://github.com/alphagov/govuk-frontend/issues/2108))
+    - create release notes for the new release (example card: [#1986](https://github.com/alphagov/govuk-frontend/issues/1986))
+    - create release notes for the new version of GOV.UK Prototype Kit (example card: [#958](https://github.com/alphagov/govuk-prototype-kit/issues/958))
+    - create a card for the new release of GOV.UK Frontend (example card: [#1987](https://github.com/alphagov/govuk-frontend/issues/1987))
+    - update the GOV.UK Design System to use the new release of GOV.UK Frontend (example card: [#1347](https://github.com/alphagov/govuk-design-system/issues/1347))
+    - create a card for the new release of GOV.UK Prototype Kit (example card: [#917](https://github.com/alphagov/govuk-prototype-kit/issues/917))
+    - update the GOV.UK Prototype Kit to use the new release (example card: [#923](https://github.com/alphagov/govuk-prototype-kit/issues/923))
+
+3. Add the issues to the [Design System sprint board](https://github.com/orgs/alphagov/projects/4).
+
+4. Content designer to:
+    - write announcements to post on Slack after we release:
+      - GOV.UK Frontend (for example, [draft comms for the cookie banner component](https://docs.google.com/document/d/1jVyMB7i94NOeflWaf3kE4Q4APMXGfluK3rOh74IHO08/edit))
+      - GOV.UK Prototype kit
+    - check who the release’s contributors are and if we have consent to include their name
+
+5. Technical writer to finalise draft of release notes and get 2i on them.
+
+6. Content designer or designer to update community backlog with rationale for any decisions we made.
+
+7. Coordinate the sign-off when you are ready to do the release. Once the team agrees, this confirms a code and content freeze. Use the [#design-system-team-channel](https://gds.slack.com/app_redirect?channel=design-system-team-channel) to confirm sign-off from:
```

Comment:
Make sure the unordered bullets start with a lower case letter, as per the style guide